.github/workflows/try-runtime.yml

```diff
@@ -0,0 +1,40 @@
+name: Run try-runtime on storage migration label
+
+on:
+  pull_request:
+    types:
+      - labeled
+
+jobs:
+  run_if_label_matches:
+    if: github.event.label.name == 'migration'
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out the repo
+        uses: actions/checkout@v3
+
+      - name: Setup worker
+        uses: "./.github/templates/setup-worker"
+
+      - name: Cache Build artefacts
+        uses: actions/cache@v3
+        with:
+          path: |
+            ~/.cargo/bin/
+            ~/.cargo/registry/index/
+            ~/.cargo/registry/cache/
+            ~/.cargo/git/db/
+            target/
+          key: ${{ runner.os }}-cargo-release-${{ env.POLKA_VERSION }}-${{ hashFiles('**/Cargo.lock') }}
+          restore-keys: ${{ runner.os }}-cargo-release-${{ env.POLKA_VERSION }}
+
+      - run: 
+          echo "Found label runtime_migration. Running tests"
+          export RUST_LOG=remote-ext=debug,runtime=debug
+          echo "---------- Running try-runtime for Trappist ----------"
+          time cargo build --locked --release -p trappist-runtime
+          time cargo run --locked --release --features try-runtime try-runtime \
```

Comment:
I am not sure if we need to use `time` command here. job logs already have timestamps.

Reply:
Agreed. Removed.